Dockerfile

```
@@ -1,8 +1,8 @@
 # https://hg.nginx.org/nginx-quic/file/tip/src/core/nginx.h
-ARG NGINX_VERSION=1.21.6
+ARG NGINX_VERSION=1.21.7
 
 # https://hg.nginx.org/nginx-quic/shortlog/quic
-ARG NGINX_COMMIT=7c2adf237091
+ARG NGINX_COMMIT=ce6d9cf0f567
```

```suggestion
ARG NGINX_COMMIT=efbcdb9b37dc
```

The latest commit from 8 June, https://hg.nginx.org/nginx-quic/rev/efbcdb9b37dc

Tagged with v1.23.0 - https://hg.nginx.org/nginx-quic/rev/6206bca52d73